frontend/src/components/GroupView/MiddleBar/index.tsx

```diff
@@ -1,25 +1,43 @@
 import React, { ReactElement } from 'react';
-import { Group, SamwiseUserProfile } from 'common/types/store-types';
+import { Group, SamwiseUserProfile, Task } from 'common/types/store-types';
 
 import GroupViewMiddleBarPeopleList from './GroupViewMiddleBarPeopleList';
 import GroupViewMiddleBarTaskQueue from './GroupViewMiddleBarTaskQueue';
 import styles from './index.module.scss';
 
 type Props = {
   readonly group: Group;
-  readonly groupMemberProfiles: readonly SamwiseUserProfile[];
+  readonly groupMemberProfiles: SamwiseUserProfile[];
```

Comment:
why are we removing readonly here

Reply:
I was getting an error

Reply:
I don't see any code in this diff that can cause this. Try to add it back and report what error you get.